.eslintrc.js

```diff
@@ -1,55 +1,57 @@
+'use strict';
+
 module.exports = {
   globals: {
     server: true,
   },
   root: true,
+  parser: 'babel-eslint',
   parserOptions: {
-    ecmaVersion: 2017,
-    sourceType: 'module'
+    ecmaVersion: 2018,
+    sourceType: 'module',
+    ecmaFeatures: {
+      legacyDecorators: true,
+    },
   },
-  plugins: [
-    'ember'
-  ],
+  plugins: ['ember'],
   extends: [
     'eslint:recommended',
-    'plugin:ember/recommended'
+    'plugin:ember/recommended',
+    // 'plugin:prettier/recommended',
```

Comment:
disabled prettier in this PR so that lint pass. Will be enabled in a separate PR as well as the all errors fixed (there are a lot of fixes, split into multiple PRs to each review)